step-capstone/package.json

```diff
@@ -9,6 +9,7 @@
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "react": "^16.13.1",
+    "react-bootstrap": "^1.0.1",
```

Comment:
why are we adding bootstrap? I though we were using material-UI as our styling

Reply:
You're right, since I just want to add a button in fast and I mistakenly wrote "Button" instead of "button", the HTML tag didn't recognize that, I went to stackoverflow and add react-bootstrap in just to test the sign-out button. I can remove it now.